lib/std/crypto/bcrypt.zig

```diff
@@ -178,6 +179,88 @@ pub fn bcrypt(
     return ct[0..dk_length].*;
 }
 
+const pbkdf_prf = struct {
+    const Self = @This();
+    pub const mac_length = 32;
+
+    hasher: Sha512,
+    sha2pass: [Sha512.digest_length]u8,
+
+    pub fn create(out: *[mac_length]u8, msg: []const u8, key: []const u8) void {
```

Comment:
Would you mind adding a short `///` comment to the new public functions and types?

Reply:
FYI this is not actually public public (the struct it is part of is private)